infra.bs

```diff
@@ -258,11 +262,36 @@ inclusive.
 
 <h3 id=strings>Strings</h3>
 
-<p>A <dfn export>string</dfn> is a sequence of <a>code points</a>. Strings are denoted by double
-quotes and monospace font.
+<p>A <dfn export>JavaScript string</dfn> is a sequence of unsigned 16-bit integers, also known as
+<dfn export lt="code unit">code units</dfn>. A <a>JavaScript string</a> can also be interpreted as
+containing <a>code points</a>, per the conversion defined in <a>The String Type</a> section of the
+JavaScript specification. [[!ECMA-262]]
+
+<p class=note>This is different from how the Unicode specification defines "code unit". In
+particular it refers exclusively to an unsigned 16-bit integer as is similar to how "code unit" is
+defined for UTF-16. However, therefore it can also refer to a surrogate that is not part of a
+surrogate pair, which is not the case for "code unit" as defined for UTF-16. [[UNICODE]]
```

Comment:
I think this note may be factually incorrect. It is the case that UTF-16 does not allow isolated surrogates. But the definition of `code unit` allows values in the surrogate range (how could it not?). The UTF-16 encoding form doesn't permit them to be isolated if the text is to be well-formed. But _Unicode 16-bit strings_ are not required to be well-formed (see my comment above)